src/styles/_catalog.scss

```diff
@@ -9,14 +13,21 @@
     display: flex;
     justify-content: center;
     align-items: center;
-    width: 200px;
+    width: $card-width;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+  @media (max-width: 1023px) {
+    grid-template-columns: repeat(3, $card-width);
+  }
+
+  @media (max-width: 767px) {
+    grid-template-columns: repeat(2, $card-width);
+  }
+
+  @media (max-width: 488px) {
```

Comment:
You could add padding bottom of 48px to this rule so the last card has some spacing between itself and bottom of the viewport

### Before
![image](https://user-images.githubusercontent.com/87986955/233134047-f4a9cf9f-fbaa-46dd-9d4e-2ee5726720b5.png)
### After
![image](https://user-images.githubusercontent.com/87986955/233134419-7dbc1100-e918-4466-9801-fa514a8eef84.png)

Reply:
I added `padding-bottom`.